project/BinCompatPlugin.scala

```diff
@@ -19,18 +19,15 @@ import com.typesafe.tools.mima.plugin.MimaPlugin
 
 object BinCompatPlugin extends AutoPlugin {
   object autoImport {
-    val binCompatVersion = settingKey[Option[String]]("version to check binary compatibility against")
   }
 
   import autoImport._
   import MimaPlugin.autoImport._
 
-  override def requires = MimaPlugin && verizon.build.ScalazPlugin
+  override def requires = MimaPlugin
 
   override def trigger = allRequirements
 
   override lazy val projectSettings = Seq(
-    mimaPreviousArtifacts := binCompatVersion.value
-      .fold(Set.empty[ModuleID])(v => Set(organization.value %% name.value % v))
```

Comment:
I'm not familiar with bincompat and Mima stuff but it seems this file existed mostly to plugin the ScalazPlugin bits. Since with this change we no longer care about that does that mean this file can be wiped?